File: java/epi/IsStringPalindromic.java
package epi;
import test_framework.EpiTest;
import test_framework.GenericTest;
public class IsStringPalindromic {
  @EpiTest(testDataFile = "../test_data/epi/is_string_palindromic.tsv")

  public static boolean isPalindromic(String s) {
    // TODO - you fill in here.
    return true;
  }

  public static void main(String[] args) {
    System.exit(
        GenericTest
            .runFromAnnotations(args, "IsStringPalindromic.java",
                                new Object() {}.getClass().getEnclosingClass())
            .ordinal());
  }
}

File: cpp/epi/is_string_palindromic.cc
#include <string>

#include "../test_framework/generic_test.h"
using std::string;

bool IsPalindromic(const string& s) {
  return false;
}

int main(int argc, char* argv[]) {
  std::vector<std::string> args{argv + 1, argv + argc};
  std::string func_name = "IsPalindromic";
  std::vector<std::string> param_names{"s"};

  return GenericTestMain(args, "is_string_palindromic.cc",
                         "../test_data/epi/is_string_palindromic.tsv", &IsPalindromic,
                         DefaultComparator{}, func_name, param_names);
}